genlib/dxfp.cpp

```diff
@@ -1475,12 +1476,18 @@ DxfToken::DxfToken()
 
 istream& operator >> (istream& stream, DxfToken& token)
 {
-   static std::string inputline;
-   stream >> inputline;
-   token.code = std::stoi(inputline);
-   stream >> token.data;
-   token.size = inputline.length() + token.data.length() + 2;   // might be missing a few end line characters --- never mind
-   return stream;
+    static std::string codeInputLine;

```

Comment:
Why static? I know it's in the original code, but is there a compelling reason to keep it? Static variables can cause all kind of shitty issues.